### PR TITLE
refactor: add geolocation to getCurrentPosition in JoinScreen.js

### PR DESCRIPTION
**Pull request by muhammadnazreen**
5b9a27c7-bc9d-437d-a3e1-ecd0bb6c24c3

### CODE REVIEW

**Comment by mangkoran** [comment]
User location can successfully retrieved. However, the codes could be refactored.

Converting to draft indefinitely.

File: screens/JoinScreen.js [hunk 1]
```diff
@@ -8,20 +8,73 @@ import {
...
+  Geolocation.getCurrentPosition(position => {
+    console.log(position);
+    setRegion({
+      region: {
+        longitude: position.coords.longitude,
+        latitude: position.coords.latitude,
+        longitudeDelta: 0.004,
+        latitudeDelta: 0.009
+      }
+    })
```
Comment (mangkoran) [inline screens/JoinScreen.js:34]: What are you trying to achieve with these lines?

Comment (muhammadnazreen) [reply screens/JoinScreen.js]: set the current location based on the getCurrentPosition? ini membolehkan dia get the position user yang sedang bergerak.

**Review by muhammadnazreen — COMMENTED** [review]
Is this one okay or not?

File: screens/JoinScreen.js (hunk 1 above)
Comment (psakurai) [reply screens/JoinScreen.js]: Sorry, tertekan

**Pull request closed without merging**